tests/utils/TestHelper.ts

```diff
@@ -358,6 +358,18 @@ async function navigateToSidebarOption(index: number): Promise<void> {
     await waitForBatchedUpdatesWithAct();
 }
 
+function assertSidebarOptionsAlphabetical() {
+    const firstElement = screen.queryByTestId('DisplayNames-0');
+    const secondElement = screen.queryByTestId('DisplayNames-1');
+    const thirdElement = screen.queryByTestId('DisplayNames-2');
+    const fourthElement = screen.queryByTestId('DisplayNames-3');
+
+    expect(firstElement).toHaveTextContent('Email Five');
```

Comment:
The text feels quite specific to the tests so not sure if this is the best to be added to the TestHelper as its no really easy to use in other tests